revolv/templates/project/project.html

```diff
@@ -13,223 +14,344 @@
+                                                        <a href="/signin/?next=/project/1/&amp;reason=donate#login" id="donate-button" class="btn-blue btn-i-want-to-donate">
+                                                            <p>DONATE</p>
+                                                        </a>
+                                                    </div>
+                                                    <!-- end .module-box -->
+                                                </aside>
+                                              </div>
+                                            </div>
+                                        </div>
+                                        <div class="main-area col-lg-8 col-md-8 col-sm-12 col-xs-12">
         {% if updates|length > 0 %}
-            <h2 class="title-blue-border title-project-updates pull-right">PROJECT UPDATES</h2>
-            <div class="clearfix"></div>
-            <div class="list-area">
+            
+                                            <h2 class="title-blue-border title-project-updates pull-right">PROJECT UPDATES</h2>
+                                            <div class="clearfix"></div>
+                                            <div class="list-area">
               {% for update in updates %}
-              <div class="row">
-                <div class="left-date">
-                  <span class="date-txt">{{ update.date|date:"o" }}</span>
-                  <span class="month-txt">{{ update.date|date:"M"|upper }}</span>
-                </div>
-                <div class="main-info">
+              
+                                                <div class="row">
+                                                    <div class="left-date">
+                                                        <span class="date-txt">{{ update.date|date:"o" }}</span>
+                                                        <span class="month-txt">{{ update.date|date:"M"|upper }}</span>
+                                                    </div>
+                                                    <div class="main-info">
                     {{ update.update_text|safe }}
-                    <ul class="social-ul">
-                      <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-fb icons"></i></a></li>
-                      <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-tw icons"></i></a></li>
-                      <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-gg icons"></i></a></li>
-                    </ul>
-                </div>
-              </div>
-              <!-- end .row -->
+                    
+                                                        <ul class="social-ul">
+                                                            <li>
+                                                                <a href="javascript:;" class="btn-icon-gray">
```

Comment:
These links to not lead anywhere. Please put in the actual URLs (ask Andreas if you do not have these).